testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -157,10 +168,21 @@ protected void insertTableRows(String tableName, String floatToDoubleType)
                         "  TINYINT '-1', " +
                         "  TINYINT '2', " +
                         "  TINYINT '-3', " +
+                        "  TINYINT '4', " +
+                        "  TINYINT '-5', " +
+                        "  TINYINT '6', " +
```

Comment:
Can we provide a two digit value for tinyint ?

Reply:
Done